miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -487,50 +487,18 @@
             </c:otherwise>
           </c:choose>
         </tr>
-        <tr>
-          <c:choose>
-            <c:when test="${empty container.locationBarcode}">
-              <td>Location:</td>
-              <td><form:input
-                  path="sequencerPartitionContainers[${containerCount.index}].locationBarcode"/></td>
-            </c:when>
-            <c:otherwise>
-              <td>Location:</td>
-              <td>
-                <span id="locationBarcode">${container.locationBarcode}</span>
-                <c:if test="${(container.securityProfile.owner.loginName eq SPRING_SECURITY_CONTEXT.authentication.principal.username)
-                                                    or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
-                  <a href="javascript:void(0);"
-                     onclick="Run.ui.editContainerLocationBarcode(jQuery('#locationBarcode'), ${containerCount.index})">
-                    <span class="fg-button ui-icon ui-icon-pencil"></span>
-                  </a>
-                </c:if>
-              </td>
-            </c:otherwise>
-          </c:choose>
-        </tr>
-        <tr>
-          <c:choose>
-            <c:when test="${empty container.validationBarcode}">
-              <td>Validation:</td>
-              <td><form:input
-                  path="sequencerPartitionContainers[${containerCount.index}].validationBarcode"/></td>
-            </c:when>
-            <c:otherwise>
-              <td>Validation:</td>
-              <td>
-                <span id="validationBarcode">${container.validationBarcode}</span>
-                <c:if test="${(container.securityProfile.owner.loginName eq SPRING_SECURITY_CONTEXT.authentication.principal.username)
-                                                    or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
-                  <a href="javascript:void(0);"
-                     onclick="editContainerValidationBarcode(jQuery('#validationBarcode'), 0)">
-                    <span class="fg-button ui-icon ui-icon-pencil"></span>
-                  </a>
-                </c:if>
-              </td>
-            </c:otherwise>
-          </c:choose>
-        </tr>
+        <c:if test="${not empty container.locationBarcode}">
+          <tr>
+            <td>Location:</td>
+            <td><span id="locationBarcode">${container.locationBarcode}</span></td>
+          </tr>
+        </c:if>
+        <c:if  test="${not empty container.validationBarcode}">
+          <tr>
+            <td>Validation:</td>
+            <td><span id="validationBarcode">${container.validationBarcode}</span></td>
+          </tr>
+        </c:if>
```

Comment:
I think the most convenient way to fill in all container data is to wait for the Run and Container to be created via notification and then add pools and location from the Run page.

This change prevents setting location on the Run page and forces the user to the Edit Container page. Fix the save instead, unless there's a reason not to?

Same applies for validationBarcode